src/protocol/internal.c

```diff
@@ -23,13 +23,18 @@
 
 #include "internal.h"
 
+#if NOISE_USE_BOLOS_VAULTAPP_BACKEND
```

Comment:
to be consistent with the library, NOISE_USE_BOLOS_VAULTAPP_BACKEND coul be renamed USE_BOLOS

Reply:
The backend is specific to the vaultApp so I would keep `USE_BOLOS_VAULTAPP_BACKEND` if that is ok to you. The NOISE prefix were there because this define is set in the vaultApp's Makefile